sdk/src/main/java/com/qonversion/android/sdk/QProductCenterManager.kt

```diff
@@ -188,6 +189,9 @@ class QProductCenterManager internal constructor(
                 } else {
                     config.uid = identityID
 
+                    // renew handled properties when userID is changed
+                    userPropertiesManager?.renewHandledProperties()
```

Comment:
Suspect, there will appear more places where these two line will be needed. There are already two places with duplicating. Let's put them into a separate function?